.isort.cfg

```diff
@@ -0,0 +1,12 @@
+[settings]
+combine_star=true
+force_sort_within_sections=true
+
+import_heading_stdlib=standard python libraries
```

Comment:
I'd like to see the first word of these capitalized (e.g. Standard python libraries), but I don't think it's critical.